### ISSUE TITLE
Big Play Button Slot ... class added

### ISSUE THREAD

**Issue by realtimeappsolutions** [quasarframework/app-extension-qmediaplayer#58]
If you use the slot for Big Play Button, the template contents still gets wrapped in a div which produces a faint white circle

```
<div class="q-media--big-button">
```

**Comment by hawkeye64** [comment]
screenshot?

**Comment by realtimeappsolutions** [comment]
![QMediaPlayer001](https://user-images.githubusercontent.com/46996440/63271625-78403b80-c292-11e9-9c1d-8027a96427e7.jpg)

This shows the white circle that gets added by the 
```
class="q-media--big-button"
```
that gets wrapped around the slot code

**Comment by realtimeappsolutions** [comment]
![QMediaPlayer002](https://user-images.githubusercontent.com/46996440/63271976-251ab880-c293-11e9-9e5f-17682a6fe237.jpg)

This shows the class="q-media--big-button" that gets wrapped around the slot code

**Comment by hawkeye64** [comment]
Yep, the circle got fixed in v1.0.6 - kind of thought that where you were going, but wasn't sure.
